service/header/sync.go

```diff
@@ -10,62 +11,119 @@ import (
 	tmbytes "github.com/tendermint/tendermint/libs/bytes"
 )
 
+// TODO:
+//  1. Sync protocol for peers to exchange their local heads on connect
+//  2. If we are far from peers but within an unbonding period - trigger sync automatically
+//  3. If we are beyond the unbonding period - request Local Head + 1 header from trusted and hardcoded peer
+//  automatically and continue sync until know head.
```

Comment:
```suggestion
//  automatically and continue sync until known head.
```
What does known mean in this context?

Reply:
TBH, I can't parse this as well from a first glance

Reply:
Those TODO are more like a personal todos, that I forgot to remove